lib/ansible/executor/action_write_locks.py

```diff
@@ -39,6 +40,8 @@
     # a strategy).  We precreate them here as an optimization
     mods = set(p['name'] for p in PKG_MGRS)
 
-    mods.update(('copy', 'file', 'setup', 'slurp', 'stat'))
+    # known modules that cause this
+    mods.update(('copy', 'file', 'slurp', 'stat', 'setup'))
+
     for mod_name in mods:
         action_write_locks[mod_name] = Lock()
```

Comment:
git checkout  lib/ansible/executor/action_write_locks.py

since there's no need for any of the changes in this file.